src/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "parserOptions": {
```

Comment:
I think we’re missing the browser env here?

Reply:
Fixed.